keyboards/momoka_ergo/keymaps/via/rules.mk

```diff
@@ -0,0 +1,9 @@
+VIA_ENABLE = yes
```

Comment:
It says to not include makefiles into the keymap, but I don't wish to toggle these flags for the default keymap. How do I go about that?

Reply:
That is talking about an actual file named `Makefile`, not `rules.mk`.